Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.11-alpine
+
+RUN apk add pipx
+
+WORKDIR template-streamlit
```

Comment:
This Dockerfile would be inherited in future streamlit-based repos, probably best to have a more generic workdir like `app`